components/contextualhelp/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@spectrum-css/contextualhelp",
+  "version": "1.0.0-alpha.0",
+  "description": "The Spectrum CSS contextualhelp component",
+  "license": "Apache-2.0",
+  "author": "Adobe",
+  "contributors": [],
+  "homepage": "https://opensource.adobe.com/spectrum-css/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/spectrum-css.git",
+    "directory": "components/contextualhelp"
+  },
+  "bugs": {
+    "url": "https://github.com/adobe/spectrum-css/issues"
+  },
+  "main": "dist/index-vars.css",
+  "scripts": {
+    "build": "gulp",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "gulp watch"
+  },
+  "peerDependencies": {
+    "@spectrum-css/actionbutton": "^3.0.20",
+    "@spectrum-css/icon": "^3.0.35",
+    "@spectrum-css/popover": "^6.0.23",
+    "@spectrum-css/tokens": ">= 7"
+  },
+  "devDependencies": {
+    "@spectrum-css/actionbutton": "^3.0.20",
+    "@spectrum-css/component-builder-simple": "^2.0.6",
+    "@spectrum-css/icon": "^3.0.35",
```

Comment:
Icon is coming in as part of actionbutton so it shouldn't need to be added separately